source/Sagan-RDBMS/PersistentObjectComparer.class.st

```diff
@@ -0,0 +1,25 @@
+Class {
+	#name : #PersistentObjectComparer,
+	#superclass : #Object,
+	#category : #'Sagan-RDBMS-Core'
+}
+
+{ #category : #asserting }
+PersistentObjectComparer >> hashFor: anObject [ 
+
+	^anObject sequentialNumber hash
+]
+
+{ #category : #asserting }
+PersistentObjectComparer >> shouldConsider: element asEqualTo: referenceElementOrProxy [
```

Comment:
element and referenceElementOrProxy are bad names
anObject, anotherObject and unproxiedObject